net-mgmt/telegraf/src/opnsense/mvc/app/models/OPNsense/Telegraf/Menu/Menu.xml

```diff
@@ -4,6 +4,7 @@
       <General order="10" url="/ui/telegraf/general/index"/>
       <Input order="20" url="/ui/telegraf/input/index"/>
       <Output order="30" url="/ui/telegraf/output/index"/>
+      <LogFile order="40" url="/diag_logs_telegraf.php"/>
```

Comment:
Can you add a visible name like here:
https://github.com/opnsense/plugins/blob/master/mail/postfix/src/opnsense/mvc/app/models/OPNsense/Postfix/Menu/Menu.xml#L12